plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-shipping-address-block/block.tsx

```diff
@@ -128,17 +125,13 @@ const Block = ( {
 		};
 	} );
 
-	// Default editing state for CustomerAddress component comes from the current address and whether or not we're in the editor.
-	const defaultEditingAddress = isEditor || ! hasAddress;
```

Comment:
I couldn't reproduce the `isEditor` need here, with a filled or empty address, it always show up expanded for me.